### PR TITLE
I Changed the unittest to 5. Please review my changes

### PR DESCRIPTION
**Pull request by dragkb**

### CODE REVIEW

**Comment by Serhii-Vorobiov** [comment]
Change to 7

File: app/src/test/java/com/example/borysgurtovyi/espressobootcampalgo/ExampleUnitTest.java [hunk 1]
```diff
@@ -23,8 +23,10 @@ public void testFizzBuzz() {
         expectedRes.add("1");
         expectedRes.add("2");
         expectedRes.add("Fizz");
-        List<String> actualResult = algo.fizzBuzz(3);
-
+        expectedRes.add("4");
+        expectedRes.add("Buzz");
+        List<String> actualResult = algo.fizzBuzz(5);
+// Create unittest to 7
```
Comment (Serhii-Vorobiov) [inline app/src/test/java/com/example/borysgurtovyi/espressobootcampalgo/ExampleUnitTest.java]: Please do the following.

**Comment by dragkb** [comment]
I changed the code

**Comment by Serhii-Vorobiov** [comment]
LGTM

**Pull request merged**